### ISSUE TITLE
Exclude Filter with multiple labels might need some work

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#4143]
Instance: dev1
SHA: 5f7d20c31
Org: LBNL Test 200
Cycle: 2018 Compliance Cycle

When using more than one label for filtering, the EXCLUDE behavior isn't necessarily intuitive. I think it might need to have the option of AND or OR applied to it. I think right now it is applying the AND option, but what if you want it to use OR? 

In my example, I don't want to see any records that have either of the labels shown below, ie, I want an OR based EXCLUDE. But what the program is giving me is an AND based EXCLUDE (I think -- my mind is having a hard time figuring this out on a Friday afternoon).

Steps to reproduce: 
* There are multiple filters on records in this cycle
![image](https://github.com/SEED-platform/seed/assets/6314950/13fb9bdf-73ec-46d7-a5b5-8d427aa172f2)
* If I filter by "DQ: HIgh Site EUI" using the "AND" option, I see the records with that label
![image](https://github.com/SEED-platform/seed/assets/6314950/fba328af-1d8a-4590-83bf-95d27f99c95a)
* If I then switch to the "EXCLUDE" option, I see all the records without that label
![image](https://github.com/SEED-platform/seed/assets/6314950/d2d8054a-9df9-4a33-b752-b9be12b0f031)
* If I use both those filters with the "AND" option, then only 2 records are displayed
![image](https://github.com/SEED-platform/seed/assets/6314950/30937c0e-1572-40c8-a96a-13000be35e34)
* With both those labels still selected as filters, if I click the EXCLUDE option, records with either of the labels are displayed.
![image](https://github.com/SEED-platform/seed/assets/6314950/e693eb31-5028-4762-8e21-0f26e5dbfb0c)

I guess this kind of makes sense if you think of the EXCLUDE as an AND query, but I was expecting the program to treat it as an "OR" query, ie, both of the filters had to not be there. 

Basically I don't want to see any records with either of the labels.

**Comment by RDmitchell** [comment]
Instance: dev1
SHA: 77b9e4afa
I am closing this issue because the filtering by label has changed radically since this issue was created. This is what it looks like now, with much more control over a complicated filter.

![image](https://github.com/SEED-platform/seed/assets/6314950/7f32a06a-f2bf-479e-9107-8b9a031aa65a)
